frontend/src/Components/LobbySettings.jsx

```diff
@@ -57,24 +57,24 @@ const LobbySettings = ({ setOpen, setOpenInfo }) => {
             <CardContent className={classes.content}>
                 <List>
                     <ListItem>
-                        <Typography style={{ marginRight: '20px' }}>Number of Mafias: </Typography>
-                        <Select style={{ color: '#E3F1F1' }} defaultValue={1} label="This is a label">
+                        <Typography style={{ marginRight: '20px', color: 'black' }}>Number of Mafias: </Typography>

```

Comment:
nitpick: would it be better to put this in a css?

Reply:
Tried but doesn't appear to be an easy fix, will leave for now